### PR TITLE
Missxa bot merge

### PR DESCRIPTION
**Pull request by missxa**
based on DE + SM + BB finals state;
Changes:
- abandoned ROS when using WildTalk -> using https://github.com/gongzhang/proc-bridge 
- improved RoboyQA (devo based):
  * reacting to jokes request
  * who are you/ who created you/ where can i see/meet you
  * no answer for general knowledge question -> avoid answer + say a random trivia fact
- more sticker emotions, 
- reduced + moved to configs telegram IO delay, 
- added a few Neo4j relations (created by + media (telegram, fb..)
- new phrases and question in verbalizer and resources,
- proper sentence capitalization in utils (using in Telegram Output already)
- added variability in botboy state (greeting)
- probability-based offering of games, 
- shuffle random list

### CODE REVIEW

File: dialog/src/main/java/roboy/talk/Verbalizer.java [hunk 1]
```diff
@@ -54,23 +54,42 @@ public Action verbalize(Interpretation interpretation){
...
+					"Hey! I was looking for someone to chat.");
 	
 	private SpeechAction greet(Interpretation interpretation){
 		return new SpeechAction(StatementBuilder.random(greetings));
 	}
 	
 	public static final RandomList<String> farewells = new RandomList<>(
-			"ciao", "goodbye", "cheerio", "bye",
-            "see you", "farewell", "bye-bye");
+			"ciao", "goodbye", "cheerio", "bye", "farewell", "bye-bye");
```
Comment (Aevyz) [inline dialog/src/main/java/roboy/talk/Verbalizer.java:110]: Is there a reason why see you is removed?

File: nlu/parser/src/main/java/edu/stanford/nlp/sempre/corenlp/FullNLPAnalyzer.java [hunk 2]
```diff
@@ -37,6 +37,7 @@
     public static class Options {
         @Option(gloss = "What CoreNLP annotators to run")
         public List<String> annotators = Lists.newArrayList(
+            "tokenize",
```
Comment (Aevyz) [inline nlu/parser/src/main/java/edu/stanford/nlp/sempre/corenlp/FullNLPAnalyzer.java:40]: Do we really need tokenize twice?

File: nlu/parser/src/main/java/edu/stanford/nlp/sempre/roboy/api/apiKeys.yml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+moviekey: xxxxxxx
```
Comment (Aevyz) [inline nlu/parser/src/main/java/edu/stanford/nlp/sempre/roboy/api/apiKeys.yml:1]: Did you move everything into Parser on purpose?

File: dialog/src/main/java/roboy/talk/Verbalizer.java (hunk 1 above)
Comment (missxa) [reply dialog/src/main/java/roboy/talk/Verbalizer.java:110]: yes, cause we want to answer "where can i see you?" which would never get through

File: nlu/parser/src/main/java/edu/stanford/nlp/sempre/corenlp/FullNLPAnalyzer.java (hunk 2 above)
Comment (missxa) [reply nlu/parser/src/main/java/edu/stanford/nlp/sempre/corenlp/FullNLPAnalyzer.java:40]: @josephbirkner @Waguramu ?
git is the witness i havent touched that file :P

**Comment by Aevyz** [comment]
Additional TODO (s): 

- [ ] When Devel/Demolution is merged into this branch and #87 is too, change all references in docs from `apikeys.yml` to `api.key`
